zlp-scheduler/app/models/user.rb

```diff
@@ -1,36 +1,45 @@
 class User < ApplicationRecord
   has_secure_password
-  validates :email, uniqueness: true
   has_many :schedules, dependent: :destroy
   has_one :cohort
-  
-  #takes care of caps/spaces inconsistency in email during registration
-    def email=(value)
-        value = value.strip.downcase
-        write_attribute :email, value
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+  validates :email, presence: true,
+                    format: { with: VALID_EMAIL_REGEX },
+                    uniqueness: true
```

Comment:
I'm not sure if this would make sense to do, but if all the forms are verifying TAMU emails, why not the user model too?

Reply:
Without the check, the admin can create users with invalid email address as below.
`()~@tamu.edu` is not a valid email address. It could be typo or some one trying to attack the website. e.g., SQL injection.

![image](https://user-images.githubusercontent.com/4607047/100050627-73e0c780-2ddf-11eb-8252-265a454d8328.png)

However, your comments reminds me again that `@user.save()` will return `false` if the form is not valid.
I will check if we handling all `@user.save()` properly.

Reply:
After considering your suggestion again, I remove the email validation in `user.rb`, and I use `email_field` to valid all user's email input. Fix in #91 .